packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -539,7 +542,8 @@ class PageView extends StatefulWidget {
 
   /// Called whenever the page in the center of the viewport changes.
   final ValueChanged<int> onPageChanged;
```

Comment:
Can you update this documentation to include a sentence about how this is different from pageDidChanged?